### ISSUE TITLE
Test - incorrect description

### ISSUE THREAD

**Issue by longnd** [ltenzil/scrape#18]
This test description isn't accurate, the expected behavior is to `return a failure error` I belive
https://github.com/ltenzil/scrape/blob/6eb163261a9102f748fc789147840f7856584ef8/spec/models/keyword_spec.rb#L76-L82

Moreover, the test should be grouped under `context`
```ruby
context "given search quota reached"
  it "returns a failure"
```

**Comment by ltenzil** [comment]
my bad. will modify it

**Comment by ltenzil** [comment]
I have copied from above test case.
it should "return quota error"
My bad

**Comment by ltenzil** [comment]
Resolved in #21

**Comment by longnd** [comment]
The fix solved this issue.
